namespaces.tf

```diff
@@ -0,0 +1,14 @@
+locals {
```

Comment:
The same thing like for the namespaces, maybe it will be better to create a file named locals.tf, for now for my perspective is ok like this. But, in the future when the `locals` vars will increase it will be very hard to find all the locals.

Reply:
I usually create the local vars in the files they are used into.